docker-compose.yml

```diff
@@ -42,7 +42,7 @@ services:
     container_name: social_mailcatcher
 
   selenium:
-    image: selenium/standalone-firefox-debug:2.48.2
+    image: selenium/standalone-firefox:2.48.2
```

Comment:
Would be great to use the `selenium/standalone-firefox-debug:2.48.2` 
Using the `-debug` container allows developers to debug tests in tools like VNC viewer

Reply:
Absolutely agree, it is nice to have it.